.github/ISSUE_TEMPLATE/defect-report.yml

```diff
@@ -0,0 +1,102 @@
+name: Defect Report
+description: File a defect report
+labels: [ "defect", "in triage" ]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for helping us in making Hyades better!
+        
+        **Please do not ask questions here!**
+        If you're not sure whether what you're experiencing is in fact a defect,
+        you can reach out to maintainers and the broader community via various channels.
+        Refer to our [contributing guidelines](https://github.com/DependencyTrack/hyades/blob/main/CONTRIBUTING.md#asking-questions) to find out where and how to ask questions.
+  - type: textarea
+    id: behavior-current
+    attributes:
+      label: Current Behavior
+      description: |-
+        Describe the current faulty behavior that you observed.
+        Consider providing screenshots, log output, and other supplementary data.
+        
+        *Files and images can be included via drag and drop into this text field.*
+    validations:
+      required: true
+  - type: textarea
+    id: steps-to-reproduce
+    attributes:
+      label: Steps to Reproduce
+      description: |-
+        Describe the exact steps of how the defect can be reproduced.
+        Consider providing screenshots, BOM files, and other supplementary data.
+        
+        *Files and images can be included via drag and drop into this text field.
+        For BOM files, please redact any internal or otherwise confidential information.*
+      value: |-
+        1.
+    validations:
+      required: true
+  - type: textarea
+    id: behavior-expected
+    attributes:
+      label: Expected Behavior
+      description: >-
+        Describe how you expect Hyades to behave instead.
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
+        ## Environment
+        Please provide some details about the environment in which you observed the defect.
+  - type: dropdown
+    id: environment-dtrack-version
+    attributes:
+      label: Hyades Version
+      description: >-
+        What version of Hyades are you using?
+      options:
+        - v0.4.0
+        - v0.3.0
+        - v0.2.0
```

Comment:
As we have multiple versions involved (hyades-apiserver, Hyades-frontend) should we change this to a text filed ?